state-chain/pallets/cf-threshold-signature/src/key_rotator.rs

```diff
@@ -147,14 +147,25 @@ impl<T: Config<I>, I: 'static> KeyRotator for Pallet<T, I> {
 						),
 					},
 				KeyRotationStatusVariant::AwaitingActivation => {
-					let new_public_key = match PendingKeyRotation::<T, I>::get() {
-						Some(KeyRotationStatus::AwaitingActivation { new_public_key }) =>
+					let (new_public_key, request_id) = match PendingKeyRotation::<T, I>::get() {
```

Comment:
to keep the naming consistent with elsewhere maybe we should rename request_id to maybe_request_id and rename tss_request_id to request_id below.